career/ladder.md

```diff
@@ -161,18 +166,16 @@ Here are some examples of other career ladders:
 
 #### Simple
 - [ ] Creates architecture that enables many potential futures without knowing exactly what the future is.
-- [ ] Understands the broad architecture of the entire system and are capable of explaining the technical decisions taken by the team.
+- [ ] Understands the broad architecture of the entire system and is capable of explaining the technical decisions taken by the team.
 
 #### Test-driven
-- [ ] Performs TDD over Test Last (TLD) nearly all the time.
-- [ ] Can speak from experience on how TDD influences class design.
 - [ ] Writes tests in multiple languages
-- [ ] Writes performance tests
+- [ ] Can speak from experience on how TDD influences class design.
 - [ ] Able to identify and be comfortable deleting tests that no longer add value
 - [ ] Thinks about [F.I.R.S.T](https://github.com/ghsukumar/SFDC_Best_Practices/wiki/F.I.R.S.T-Principles-of-Unit-Testing) tests
```

Comment:
Dead link. Might use this instead: https://tasdikrahman.me/2019/03/13/f-i-r-s-t-principles-of-testing/